CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v0.22.1
+
+### Fixed
+
+- Moved metallb error recording inside context switch, to avoid
```

Comment:
add pr link?

Reply:
Gimme some hook magic that automatically replaces a template string with the PR number. We're kinda lax on it, especially for single-change releases. Ultimately blame can usually tell you, unless someone else added the entry retroactively.